spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -91,6 +91,99 @@
         end
       end
 
+      context 'reuses valid capture app urls when appropriate', allow_browser_log: true do
```

Comment:
for the specs below, we only have 1 capture app url ... how do we know that the url is not changing from one url to another when the value assigned in our spec is always the same? 🤔